CRT.c

```diff
@@ -1046,7 +1046,7 @@ void CRT_init(const Settings* settings, bool allowUnicode, bool retainScreenOnEx
       CRT_scrollHAmount = 5;
    }
 
-   if (termType && (String_startsWith(termType, "xterm") || String_eq(termType, "vt220"))) {
+   if (termType && (String_startsWith(termType, "xterm") || String_eq(termType, "vt220") || String_startsWith(termType, "screen") || String_startsWith(termType, "tmux") || String_startsWith(termType, "st"))) {
```

Comment:
```suggestion
   if (termType && (
      String_startsWith(termType, "xterm") ||
      String_eq(termType, "vt220") ||
      String_startsWith(termType, "screen") ||
      String_startsWith(termType, "tmux") ||
      String_startsWith(termType, "st")
   )) {
```

Isn't `st` a bit generic on its own?

Reply:
I agree `st` is pretty generic. On my system, the termtypes that are erroneously matched are as follows
|TERM|description|
|---|--|
|`st52`|Atari ST|
|`st52-old`|Atari ST with VT52 emulation|
|`stv52`|MiNT virtual console|
|`stv52pc`|MiNT virtual console with PC charset|

The default TERM for ST is set to `st-256color` so we could use that instead.

Reply:
I'd prefer that as `st` (Suckless Terminal) and `st52` (Atari ST) are quite different and I doubt the Atari ST implements most of the features relevant in that part of the code. Not sure about `stv52` though, but from what I could gather this seems kinda limited in its feature set too. `String_eq(termType, "st") || String_beginsWith(termType, "st-")` might be a compromise here to capture both `st` and `st-256color` (or variants) …

Also a quick note about grouping checks, as this might be of interest here: We could do something like:

```c
      (termType[0] == 's' && (
         String_beginsWith(…, 'screen') ||
         String_eq(…, 'st') ||
         String_beginsWith(…, 'st-')
      )) ||
```

For groups with at least 3 items as this allows the compiler to short-circuit on large portions of the condition with similar first letter.

Reply:
Great idea! Short circuiting is more efficient, but the former you provided is a simper implementation. Which one do you prefer we use?

Reply:
I suggest you to factor this into its own function first, since the list of string would be eventually growing anyway, that would make the list more readable and manageable.

Reply:
I think that would be a good idea. I'll refactor the code.